introduction.md

```diff
@@ -1 +1,7 @@
-Hi I'm Casper, and am currently feeling really bad about making too short introductions. My favorite track is definitely the JavaScript track.
\ No newline at end of file
+Hi I'm Casper, and am currently feeling really bad about making too short introductions. My favorite track is definitely the JavaScript track.
+
+* Git
+* JavaScript
+* D3
+* Python
+* .Net
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?